uPortal-webapp/src/main/webapp/media/skins/respondr/common/less/navigation.less

```
@@ -390,6 +390,76 @@
           -o-transform: none !important;
           transform: none !important;
         }
+        div#portalNavigationTabGroup {
```

is there only one `portalNavigationTabGroup`?
If so
```suggestion
        #portalNavigationTabGroup {
```

would be unique enough